src/Trixi.jl

```diff
@@ -245,58 +257,57 @@ export DGMulti, DGMultiBasis, estimate_dt, DGMultiMesh, GaussSBP
 export ViscousFormulationBassiRebay1, ViscousFormulationLocalDG
 
 # Visualization-related exports
-export PlotData1D, PlotData2D, ScalarPlotData2D, getmesh, adapt_to_mesh_level!, adapt_to_mesh_level,
+export PlotData1D, PlotData2D, ScalarPlotData2D, getmesh, adapt_to_mesh_level!,
+       adapt_to_mesh_level,
        iplot, iplot!
 
 function __init__()
-  init_mpi()
+    init_mpi()
 
-  init_p4est()
+    init_p4est()
 
-  register_error_hints()
+    register_error_hints()
 
-  # Enable features that depend on the availability of the Plots package
-  @require Plots="91a5bcdd-55d7-5caf-9e0b-520d859cae80" begin
-    using .Plots: Plots
-  end
+    # Enable features that depend on the availability of the Plots package
+    @require Plots="91a5bcdd-55d7-5caf-9e0b-520d859cae80" begin
```

Comment:
Unfortunately, this is currently not supported by JuliaFormatter. It would be interesting though, so if you decide to create an issue with JuliaFormatter, please cc me

Reply:
I created an issue, https://github.com/domluna/JuliaFormatter.jl/issues/737.

Reply:
Thanks!